### DISCUSSION TITLE
using scenes

### DISCUSSION THREAD

**Discussion by ineall** [krahabb/meross_lan#150] [Q&A] [answered]
First a big thanks for this integration. 
I'm really new to HA and maybe that is my problem. I have a msh300 hub and 5 trvs. I have 3 radiators in my kitchen. I can control them all individually through the integration with no problems but I'd like to control them as a group. So though I should define a scene that I could use to turn on the rads when the auto schedule says they should be off. But I just don't understand the UI that describes a scene. The ui says to select the devices or entities and set there properties, but after selecting a trv  entity I see 4 properties listed but there is no ui to say what state I want. Tapping a property takes me to the entity details but still doesn't allow me to say which property I want in which state. I feel like I am missing something about what scenes are for. Could someone explain what my misunderstanding is?
Thanks very much
Ian

**Comment by krahabb** [comment, marked as answer]
Hello @ineall ,
I'm sorry but I'm also totally unaware of how scenes work and I guess you could find better support on the general HA support forums. I'll let this open anyway if someone else has a quick solution for your issue :)

**Reply by ineall** [reply]
After reading a few of the forum posts and reading the help page for the scenes editor I understand what is going on now.  Because of the time it can take for a change to a TRV made via the meross_lan integration (or the meross App) to be seen at the TRV itself I find the scene editor is best used in the following way (if including TRVs).
- Set the TRVs to the state you want the scene to reflect. Do this on the device or through the app/integration but make sure you leave long enough for the state changes to be really made at the TRV and for the app/integration to reflect them. 
- Enter the scene editor UI and if you had previously added your TRVs, delete them from the scene and save the scene and exit the UI and re-enter the scene editor. 
- select your scene to edit, add the TRVs, save and exit the scene editor. 
i.e. the scene editor will capture the current state of the devices that you add. 
The help page says you can change the state in the UI and the changed state will be captured, and then on exit from the UI the original state will be restored. That may be true in the general case but I couldn't get it to work and my scenes just seemed to capture a jumble of states.  Using the above long hand method to define the scene on the devices first and then create the scene to capture the states works well for me.

**Reply by krahabb** [reply]
I see your point. I remember too reading about scenes they work like a 'snapshot' (which you would then be able to lately restore to the same state)
Effectively the TRV has a delay when receving a command either from its own app or from meross_lan (this delay is not visible if you 'physically' operate the device) and this is stated in the official documentation about the valve. It is meant to avoid excessive battery usage/drain when you're maybe sending multiple settings back to back.
